test/integration/api_docs/services_controller_test.rb

```diff
@@ -105,11 +104,11 @@ def test_show_onprem_account_management_api
       select_endpoint = Proc.new { |api| api['path'] == '/admin/api/account_plans/{id}.xml' }
       ThreeScale.config.stubs(onpremises: false)
       get '/api_docs/services/account_management_api.json'
-      assert_not_empty JSON.parse(response.body)['apis'].select(&select_endpoint)
+      assert_not_empty JSON.parse(response.body)['paths'].keys
```

Comment:
Same as above. At line 104:

`select_endpoint = Proc.new { |api| api == '/admin/api/account_plans/{id}.xml' }`

Then here:

```suggestion
      assert_not_empty JSON.parse(response.body)['paths'].select(&select_endpoint)
```